lib/matplotlib/collections.py

```diff
@@ -828,6 +818,11 @@ def set_edgecolor(self, c):
             The collection edgecolor(s).  If a sequence, the patches cycle
             through it.  If 'face', match the facecolor.
         """
+        # We pass through a default value for use in LineCollection.
+        # This allows us to maintain None as the default indicator in
+        # _original_edgecolor.
+        if isinstance(c, str) and c.lower() in ("none", "face"):
```

Comment:
Do we want to extend `cbook._str_lower_equal` to support multiple values

`cbook._str_lower_equal(c, ("none", "face"))`?

Reply:
I'm not motivated to do so.  I'm not a fan of cbook._str_lower_equal as a replacement for a simple one-liner.